components/script/webdriver_handlers.rs

```diff
@@ -381,12 +381,9 @@ fn get_element_in_view_center_point(element: &Element) -> Option<Point2D<i64>> {
         .GetBody()
         .map(DomRoot::upcast::<Element>)
         .and_then(|body| {
-            element
-                .GetClientRects()
-                .iter()
-                // Step 1
-                .next()
-                .map(|rectangle| {
+           let rectangle_element = element.GetClientRects();
+           let rectangle_option = rectangle_element.iter().next();
+           rectangle_option.map(|rectangle| {
```

Comment:
I believe that @sagudev meant that you should keep the comment. I recommend something like you had before, just restoring the original comment. We could even improve it a bit:

```
        // Step 1: Let rectangle be the first element of the DOMRect sequence 
        // returned by calling getClientRects() on element.
        element
            .GetClientRects().first()
            .map(|rectangle| {
```